crypto/sm4/asm/sm4-riscv64-zvksed.pl

```diff
@@ -59,6 +59,524 @@
 .text
 ___
 
+my ($V0, $V1, $V2, $V3, $V4, $V5, $V6, $V7,
+    $V8, $V9, $V10, $V11, $V12, $V13, $V14, $V15,
+    $V16, $V17, $V18, $V19, $V20, $V21, $V22, $V23,
+    $V24, $V25, $V26, $V27, $V28, $V29, $V30, $V31,
+) = map("v$_",(0..31));
+
+# Load 32 round keys to v1-v8 registers.
+sub enc_load_key {
+    my $keys = shift;
+
+    my $code=<<___;
+    @{[vsetivli "zero", 4, "e32", "m1", "ta", "ma"]} 
+    @{[vle32_v $V16, $keys]} # rk[0:3]  
```

Comment:
My suggestion would be using a more consistent naming style as the rest of this file like the one in `rv64i_zvksed_sm4_encrypt`
```
    @{[vsetivli__x0_4_e32_m1_tu_mu]}

    # Order of elements was adjusted in set_encrypt_key()
    @{[vle32_v $vk0, $keys]} # rk[0:3]
```

Also, since you extracted the `enc_load_key` subroutine, you might also simplify the `rv64i_zvksed_sm4_encrypt` code.